.changesets/fix_garypen_router_411_batching_entity_caching.md

```diff
@@ -0,0 +1,5 @@
+### Allow query batching and entity caching to work together ([PR #5598](https://github.com/apollographql/router/pull/5598))
+
+Without co-ordination, entity caching and subgraph batching will not work together. This changes entity caching so that, if a subgraph request is identified as being part of a batch, entity caching is not applied to the request.
```

Comment:
@garypen - please review

```suggestion
### Allow query batching and entity caching to work together ([PR #5598](https://github.com/apollographql/router/pull/5598))

The router now supports entity caching and subgraph to run simultaneously. Specifically, this change updates the entity cache to not cache a subgraph request if the request is part of a batch.

so that, if a subgraph request is identified as being part of a batch, entity caching is not applied to the request.
```

Reply:
I think there's a "batching" missing in the middle of line 3, just after "subgraph".

In the second sentence, I think this is clearer:

Specifically, this change updates entity caching to ignore a subgraph request if the request is part of a batch.

With that in place, I don't think we need lines 4 and 5. (also line 5 seems to be missing something from the start of the line as is).

Reply:
Thanks @garypen, updating as suggested:

```suggestion
### Allow query batching and entity caching to work together ([PR #5598](https://github.com/apollographql/router/pull/5598))

The router now supports entity caching and subgraph batching to run simultaneously. Specifically, this change updates entity caching to ignore a subgraph request if the request is part of a batch.
```